### ISSUE TITLE
Conversation list floating action button's symbol is ambiguous

### ISSUE THREAD

**Issue by RiseT** [WhisperSystems/Signal-Android#5226]
The graphical symbol (pencil) of the **floating action button in the conversation list** doesn't make much sense.

A **pencil** usually represents **writing**. But tapping that floating action button opens Signal's **contact list** (and is btw currently the only way to achieve that), which is not very intuitive.

"Writing" in that context could be interpreted as e. g.
a) writing a new message within an existing conversation (i. e. in the conversation list) or
b) starting a new conversation (i. e. in the contact list).

One the other hand, a **plus sign** ("+") represents **adding**, like in **adding a new conversation**.
Within the context of the conversation list, I'm not aware of any other imaginable interpretations, so a plus sign is not ambiguous regarding its purpose.

So in conclusion, **I suggest that the pencil symbol is replaced by a plus sign.**
An alternative possibility would be using that **"contacts" symbol** pictured below (after all, tapping the button opens Signal's contact list).

CC: @agrajaghh

![patterns_actions_fab_qualities1](https://cloud.githubusercontent.com/assets/12624674/13033598/38b4280e-d31d-11e5-99e9-07088f0235be.jpg)

**Comment by johanw666** [comment]
It's not that unfamiliar. Telegram works the same.

**Comment by RiseT** [comment]
Just looked it up. BBM seems to use a text bubble in combination with a plus sign.
But even if Telegram uses a pencil as well - that doesn't change the ambiguity of a pencil in that context.

**Comment by moxie0** [comment]
it used to be a plus, and we intentionally switched it to a pencil
